packages/relay/src/lib/eth.ts

```diff
@@ -1643,9 +1643,11 @@ export class EthImpl implements Eth {
 
     const selector = getFunctionSelector(call.data!);
 
-    const shouldForceToConsensus =
-      process.env.ETH_CALL_FORCE_TO_CONSENSUS_BY_SELECTOR == 'true' &&
-      constants.ETH_CALL_SELECTORS_ALWAYS_TO_CONSENSUS.indexOf(selector) !== -1;
+    // When eth_call is invoked with a selector listed in specialSelectors, it will be routed through the consensus node, regardless of ETH_CALL_DEFAULT_TO_CONSENSUS_NODE.
+    // note: this feature is a workaround for when a feature is supported by consensus node but not yet by mirror node.
+    // Follow this ticket https://github.com/hashgraph/hedera-json-rpc-relay/issues/2984 to revisit and remove special selectors.
+    const specialSelectors = process.env.ETH_CALL_CONSENSUS_SELECTORS || [''];
```

Comment:
This value will have different types, it can be either `string` or `string[]` (when `ETH_CALL_CONSENSUS_SELECTORS` is not defined. Is this intended?